pyproject.toml

```diff
@@ -165,6 +165,13 @@ h5netcdf = ">=1.5.0,<2"
 [tool.pixi.feature.icechunk-dev.dependencies]
 rust = "*"
 
+[tool.pixi.feature.minimum-versions.dependencies]
+xarray = "==2025.3.0"
+numpy = "==2.0.0"
+numcodecs = "==0.15.1"
+zarr = "==3.0.8"
+obstore = "==0.5.1"
```

Comment:
These are duplicated with the `dependencies` section above, but it seems that references to other values are [forbidden in TOML](https://github.com/toml-lang/toml/issues/77#issuecomment-14274593) so that's probably unavoidable.